libs/net/template/ReportExtensions.cs

```diff
@@ -370,12 +370,6 @@ public static string GetSource(this ContentModel content, ReportEngineContentMod
         var source = (context.Settings.Headline.ShowSource
                 ? content?.Source?.Name
                 : "");
-        // source is empty or empty string, use the other source
-        if (String.IsNullOrWhiteSpace(source))
```

Comment:
The problem here is that `OtherSource` should be displayed when content does not have a `Source`.  The reason `OtherSource` exists is because editors can add content that doesn't not belong to any of the currently configured sources.

Reply:
Wouldn't we then do the following?
When `Show Source == false` do not show `Content.Source.Name` or `Content.OtherSource`.
When `Show Source == true` show `Content.Source?.Name ?? Content.OtherSource`.